content/en/docs/concepts/workloads/controllers/deployment.md

```diff
@@ -399,7 +399,7 @@ rolled back.
 * Press Ctrl-C to stop the above rollout status watch. For more information on stuck rollouts,
 [read more here](#deployment-status).
 
-* You see that the number of old replicas (`nginx-deployment-1564180365` and `nginx-deployment-2035384211`) is 2, and new replicas (nginx-deployment-3066724191) is 1.
+* You see that the number of old replicas (`nginx-deployment-1564180365` and `nginx-deployment-1564180365`) is 3, and new replicas (nginx-deployment-3066724191) is 1.
```

Comment:
The below part of proposed change is not correct:

>  (`nginx-deployment-1564180365` and `nginx-deployment-1564180365`)

Looking at the previous section of same page I see in the example `nginx-deployment-2035384211` is the old replicaSet that was initial updated resulting it being scaled down to 0 and `nginx-deployment-1564180365` scaled to 3. Hence I feel the name of replicaSet was correct as before the change. I have made suggestion for the same change and rest of it looks fine.

Deploy Preview: [deploy-preview-40336](https://deploy-preview-40336--kubernetes-io-main-staging.netlify.app/docs/concepts/workloads/controllers/deployment/)